sklearn/feature_extraction/tests/test_text.py

```diff
@@ -417,13 +417,13 @@ def test_vectorizer():
         assert_equal(counts_test[0, vocabulary["water"]], 1)
 
         # stop word from the fixed list
-        assert_false("the" in vocabulary)
+        assert not("the" in vocabulary)
 
         # stop word found automatically by the vectorizer DF thresholding
         # words that are high frequent across the complete corpus are likely
         # to be not informative (either real stop words of extraction
         # artifacts)
-        assert_false("copyright" in vocabulary)
+        assert not("copyright" in vocabulary)
```

Comment:
`"copyright" not in vocabulary` here and below.

The only reason this doesn't raise flake8 warning is that of parenthesis,
```py
assert not "copyright" in vocabulary
```
produces
```
test for membership should be 'not in' [E713]  
```